core/benchmarks/ZStandardBenchmark-results.txt

```diff
@@ -2,48 +2,48 @@
 ------------------------------------------------------------------------------------------------------------------------
-Parallel Compression with 0 workers                  61             62           0          0.0      475889.8       1.0X
-Parallel Compression with 1 workers                  43             45           2          0.0      338238.7       1.4X
-Parallel Compression with 2 workers                  42             45           2          0.0      327766.3       1.5X
-Parallel Compression with 4 workers                  43             45           1          0.0      334407.0       1.4X
-Parallel Compression with 8 workers                  44             47           1          0.0      345047.0       1.4X
-Parallel Compression with 16 workers                 50             52           1          0.0      387573.7       1.2X
+Parallel Compression with 0 workers                  78             78           0          0.0      607618.7       1.0X
+Parallel Compression with 1 workers                  57             58           3          0.0      441422.6       1.4X
+Parallel Compression with 2 workers                  47             49           2          0.0      366700.6       1.7X
+Parallel Compression with 4 workers                  42             43           2          0.0      325502.2       1.9X
+Parallel Compression with 8 workers                  44             46           1          0.0      342866.0       1.8X
+Parallel Compression with 16 workers                 48             50           1          0.0      371828.3       1.6X
 
-OpenJDK 64-Bit Server VM 17.0.14+7-LTS on Linux 6.8.0-1020-azure
+OpenJDK 64-Bit Server VM 17.0.14+7-LTS on Linux 6.8.0-1021-azure
 AMD EPYC 7763 64-Core Processor
 Parallel Compression at level 9:          Best Time(ms)   Avg Time(ms)   Stdev(ms)    Rate(M/s)   Per Row(ns)   Relative
 ------------------------------------------------------------------------------------------------------------------------
-Parallel Compression with 0 workers                 161            163           1          0.0     1260900.1       1.0X
-Parallel Compression with 1 workers                 192            194           3          0.0     1501584.9       0.8X
-Parallel Compression with 2 workers                 116            119           2          0.0      908607.3       1.4X
-Parallel Compression with 4 workers                 112            116           6          0.0      878265.0       1.4X
-Parallel Compression with 8 workers                 118            121           3          0.0      921810.4       1.4X
-Parallel Compression with 16 workers                115            120           3          0.0      899718.4       1.4X
+Parallel Compression with 0 workers                 260            260           1          0.0     2027796.3       1.0X
+Parallel Compression with 1 workers                 287            289           3          0.0     2245048.0       0.9X
```

Comment:
The degradation looks obviously. cc @yaooqinn

Reply:
Can you run these benchmarks one or two times more? @pan3793

Reply:
3 times on this PR branch
- https://github.com/pan3793/spark/actions/runs/13496989883
- https://github.com/pan3793/spark/actions/runs/13496995223
- https://github.com/pan3793/spark/actions/runs/13496999794

3 times on master branch
- https://github.com/pan3793/spark/actions/runs/13497001661
- https://github.com/pan3793/spark/actions/runs/13497002797
- https://github.com/pan3793/spark/actions/runs/13497003849